src/webgpu/shader/execution/expression/unary/ai_complement.cache.ts

```diff
@@ -0,0 +1,11 @@
+import { abstractInt } from '../../../../util/conversion.js';
+import { fullI64Range } from '../../../../util/math.js';
+import { makeCaseCache } from '../case_cache.js';
+
+export const d = makeCaseCache('unary/ai_complement', {
```

Comment:
Having a case cache for integer tests like this seems like it would be a pessimism. I would strongly suspect that the overhead for deserializing these cases would far outweigh the cost of runtime generation.

Reply:
The existing integer complement tests use this pattern. Should I convert them away from using the cache too?

Reply:
Given that there's no heavy interval logic being precomputed, that seems like a good idea.

Reply:
Done